features/users.feature

```diff
@@ -14,6 +14,6 @@ Feature: Check users page/tab
         And I should see a "Active" link in the left menu
         And I should see a "Deactivated" link in the left menu
         And I should see a "All" link in the left menu
-        #And I should see a "admin" link in "td" "first-column "
-        And I should see 3 "admin" links
+        And I should see 2 "admin" links
```

Comment:
I would replace the admin links checks with https://github.com/SUSE/spacewalk-testsuite-base/pull/15